src/TMediaData.h

```diff
@@ -0,0 +1,90 @@
+#ifndef MUDLET_TMEDIA_DATA_H
+#define MUDLET_TMEDIA_DATA_H
+
+/***************************************************************************
+ *   Copyright (C) 2008-2013 by Heiko Koehn - KoehnHeiko@googlemail.com    *
+ *   Copyright (C) 2014-2017 by Ahmed Charles - acharles@outlook.com       *
+ *   Copyright (C) 2014-2019 by Stephen Lyons - slysven@virginmedia.com    *
```

Comment:
Given the substantial changes you are making can we also have your name in the relevant places - so every knows who to ~~blame~~*praise*. :laughing:

Reply:
There's another PR with these changes. I didn't change the code, I just moved it to a different file to remove dependency from the MXP code to the MediaPlayer and make the tests compile.